webpack.config.js

```diff
@@ -8,6 +8,9 @@ const CopyWebpackPlugin = require( 'copy-webpack-plugin' );
 const { DefinePlugin } = require( 'webpack' );
 const WebpackRTLPlugin = require( 'webpack-rtl-plugin' );
 const FixStyleOnlyEntriesPlugin = require( 'webpack-fix-style-only-entries' );
+const BundleAnalyzerPlugin = require( 'webpack-bundle-analyzer' )
```

Comment:
I was getting really inconsistent results with webpack-bundle-analyzer (as it, many code end up being unmapped) so I switched to source-map-analyzer, did you get a similar issue?

Reply:
I didn't notice any problems like that with webpack-bundle-analyzer.